modules/wc-product-editor/module.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Elementor\Modules\WcProductEditor;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Plugin;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
+		add_action( 'admin_footer', [ $this, 'print_button_js_template' ] );
+	}
+
+	public static function is_active() {
+		return self::is_new_woocommerce_product_editor_page();
+	}
+
+	public function enqueue_assets() {
+		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
+		wp_enqueue_script( 'elementor-wc-product-editor', ELEMENTOR_ASSETS_URL . 'js/wc-product-editor' . $suffix . '.js', [], ELEMENTOR_VERSION, true );
```

Comment:
Can you also break `wp_enqueue_script()` into several lines for readability? And add a space between this and `wp_enqueue_style()`